src/components/Form.js

```diff
@@ -122,6 +122,7 @@ class Form extends React.Component {
      * @returns {React.Component}
      */
     childrenWrapperWithProps(children) {
+        // eslint-disable-next-line rulesdir/prefer-underscore-method
         return React.Children.map(children, (child) => {
```

Comment:
This is not related to the issue.

@luacmartins any reason `_.map()` wasn't used here instead?

Reply:
@rushatgabhane  It doesn't pass lint check. I guess it's not a big deal, thus I added it. 
Reference: https://reactjs.org/docs/react-api.html#reactchildrenmap

Reply:
@rushatgabhane great question! `React.Children.map` is not equivalent to `_.map`, as it's a special method provided to React's children.

Reply:
Thanks so much!

Reply:
@phivh Something is off as I don't see that comment on main https://github.com/Expensify/App/blob/main/src/components/Form.js#L125. Make sure to pull the latest changes since we added that rule to the linter here - https://github.com/Expensify/eslint-config-expensify/pull/44

Reply:
yeah, it couldn't have been merged to main if it didn't pass the lint check